### ISSUE TITLE
Add user authentication UI component

### ISSUE THREAD

**Issue by jdmedlock** [chingu-voyages/v33-bears-team-10#35]
**__User Story Description__**
As a end user
I want to be prompted for my credentials
So I can be authenticated to the application to allow it to personalize my experience using my Spotify data

**__Steps to Follow (optional)__**
- [ ] TBD

**__Additional Considerations__**
TBD

**Comment by angelopoole** [comment]
I've gone through an ideation phase on how to implement this, alot of it would be put into action once I have a mock of the server up on my machine.

Planning im on contacting Hunter to ask about what specific env variables I'll need and setup for my local Machine. 

Once I have that done I can start coding the redirection and basic application state.
Redux vs context is hard ATM, I'm thinking using context may be the decision here and with that I can just use a useAuth hook.

**Comment by jdmedlock** [comment]
During September 16 Open Topic session the question came up regarding how does Spotify know if authentication is required? Is a cookie maintained for this? @angelopoole will be researching this and reaching out to others if he needs help.

**Comment by angelopoole** [comment]
With hunters help, I've set up the backend server locally on my computer, I've gotten to the redirect section and am going to figure out how the redirect interacts with the frontend. Should be able to have this complete by tomorrow.

In response to the September 16th open topic, I'm still researching the cookie bit but it looks like I'd just grab the o-auth token and save it to cookies, something along those lines. 

We'd Then re-auth with passport. that's my general line of thinking but ill definitely get back to you on this when I've fiddled with it a bit more.
